Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2675,30 +2675,30 @@ namespace OpenInfraPlatform
 						return getDirectionOfCurve<TCurve>(curve, trimming.get<1>());
 					}
 					default:
-						throw oip::InconsistentGeometryException(curve, "TrimmingSelect is wrong!");
+						throw oip::InconsistentGeometryException(curve, "IfcCurveMeasureSelect  is wrong!");

```

Comment:
This is a mistake on my side - this function switches the possibilities of an `IfcTrimmingSelect`, so the previous error was correct.

Reply:
Ok, I will correct it

Reply:
Resolved: 3ba5478